### DISCUSSION TITLE
Inquiry about using stylesheet

### DISCUSSION THREAD

**Discussion by baesumin** [nativewind/nativewind#360] [Q&A] [answered]
hello!
Will there be any performance problems if used in combination with stylesheet?
The reason for using them together is that there are times when stylesheets are useful.

**Comment by marklawlor** [comment, marked as answer]
There are no issues with mixing the two together.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
